components/domains/steps/userInfoForm.tsx

```diff
@@ -154,17 +154,12 @@ const UserInfoForm: FunctionComponent<UserInfoFormProps> = ({
         </div>
         <div className={styles.summary}>
           <div>
-            {address ? (
```

Comment:
Keep the address check, so that if the wallet is not. connected, it shows the ConnectButton